### PR TITLE
Feat/contactus

### PR DESCRIPTION
**Pull request by boneycd**

### CODE REVIEW

File: src/components/ContactUsForm.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          className="m-auto w-full px-4 sm:max-w-[500px] sm:px-12"
+        >
+          <h1 className="text-3xl font-semibold mb-3 sm:text-4xl text-center">
+            Contact Us
+          </h1>
+          
+          
+          <TextInput
+            type="text"
+            name="username"
```
Comment (suanTech) [inline src/components/ContactUsForm.jsx]: Please change name and id to match the field

File: src/components/ContactUsForm.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            id="username"
+            label="Subject*"
+            onChange={handleChange}
+            required
+          />
+          
+          
+          {error && error.field === "password" ? (
```
Comment (suanTech) [inline src/components/ContactUsForm.jsx]: remove this part

File: src/components/ContactUsForm.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          )}
+          <TextInput
+            type="text"
+            name="username"
+            id="username"
+            label="Subject*"
+            onChange={handleChange}
+            required
+          />
+
```
Comment (suanTech) [inline src/components/ContactUsForm.jsx]: add a body with TextInput size="md"

**Review by suanTech — CHANGES_REQUESTED** [review]

File: src/components/ContactUsForm.jsx (hunk 1 above)
Comment (boneycd) [reply src/components/ContactUsForm.jsx]: Thanks Suan. Its done

File: src/components/ContactUsForm.jsx (hunk 2 above)
Comment (boneycd) [reply src/components/ContactUsForm.jsx]: Thanks Suan. Its done

File: src/components/ContactUsForm.jsx (hunk 3 above)
Comment (boneycd) [reply src/components/ContactUsForm.jsx]: Thanks Suan , Not sure if i did it right.

**Comment by boneycd** [comment]
Formspree

File: src/components/ContactUsForm.jsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+            href="/login"
+            size="md"
+            buttonStyle="secondary"
+          >
+            Login
+          </Button>
+        </SuccessfulCard>
+      ) : formState === "error" ? (
+        // Todo: handle exceptions
+        <p>Error while submitting the sign up form</p>
```
Comment (suanTech) [inline src/components/ContactUsForm.jsx]: Please change the successful message, something like "Thanks for contacting us, we'll get back to you shortly."
And also button should navigate to the home page, with Go back to home text.

**Review by suanTech — CHANGES_REQUESTED** [review]
Great work Boney!! Just need a little change on texts.

**Review by suanTech — APPROVED** [review]
Nice! Merge it in!

**Pull request merged**